Resources/Prototypes/Entities/Clothing/Shoes/butcherableshoes.yml

```diff
@@ -0,0 +1,10 @@
+- type: entity
+  abstract: true
+  parent: ClothingShoesBase
+  id: ButcherableShoes
```

Comment:
put this in `base_clothingshoes.yml` and change the prototype ID to `ClothingShoesBaseButcherable` to keep it in line with the other clothing IDs.